tensorflow/python/ops/image_ops_impl.py

```diff
@@ -356,6 +356,12 @@ def random_flip_up_down(image, seed=None):
     A tensor of the same type and shape as `image`.
   Raises:
     ValueError: if the shape of `image` not supported.
+    
+  Usage Example:
+    ```python
+    >> x = tf.random.normal(shape=(256, 256, 3))
+    >> tf.image.random_flip_up_down(x)
```

Comment:
Would be nice to convert all these examples to doctests: use simple inputs (2 by 2 for example, nice values), and then print the output at the end.

Reply:
Hello, I finally added the example outputs. Could you please now check my PR again?